modules/nixos/services/bobaboard/default.nix

```diff
@@ -126,8 +189,11 @@ in
       # Remove ExecStart when enabling this
       script = ''
         if ! [ -f /var/lib/bobaboard/.migrate ]; then
+          ${bobabackend-packages.bobadatabase-init}/bin/bobadatabase-init
+          if ${boolToString cfg.database.seed}; then
```

Comment:
Rather than interpolating true/false and executing this code in bash, let's check whether we want to run this command in Nix and only add it to the script if we do. I'm on mobile atm, but I would choose to use `lib.optionalString`. There are some examples of this in the writefreely module.

Reply:
Ok, I *think* I did what you asked me :)